test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -158,8 +158,8 @@ Describe "PowerShellGet - Module tests (Admin)" -tags @('Feature', 'RequireAdmin
         Remove-InstalledModules
     }
 
-    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Travis CI Linux build
-    It "Should install a module correctly to the required location with AllUsers scope" -Pending:$IsLinux {
+    ## Marked as 'Pending' on Linux for now because the test requires root privilege but we cannot do it now in our Azure DevOps Linux build
+    It "Should install a module correctly to the required location with AllUsers scope" -Tags @('RequireSudoOnUnix') {
```

Comment:
Tags is for Describe block.

Reply:
How would I add tags here?

Reply:
Please remove the tag from "It".
Line 148 should be:
```
-Tags @('Feature','RequireAdminOnWindows', 'RequireSudoOnUnix'
```